css/pages/index.css

```diff
@@ -847,9 +865,10 @@ footer {
     font-size: 15px;
   }
 
-  #checkout-tag {
+  /* #checkout-tag {
     font-size: 1em;
-  }
+  }*/
```

Comment:
If we are not using this. Can we please remove it ?

Reply:
Removed commented code